src/test/java/trackr/logic/LogicManagerTest.java

```diff
@@ -71,11 +72,11 @@ public void execute_validCommand_success() throws Exception {
     @Test
     public void execute_storageThrowsIoException_throwsCommandException() {
         // Setup LogicManager with JsonAddressBookIoExceptionThrowingStub
-        JsonAddressBookStorage addressBookStorage =
-                new JsonAddressBookIoExceptionThrowingStub(temporaryFolder.resolve("ioExceptionAddressBook.json"));
+        JsonTrackrStorage trackrStorage =
+                new JsonTrackrIoExceptionThrowingStub(temporaryFolder.resolve("ioExceptionAddressBook.json"));
```

Comment:
The string should be "ioExcpetionTrackr.json" since we are talking about trackrStorage.